libctl/meep.scm.in

```diff
@@ -1030,10 +1024,7 @@
 			   (if (null? dt)
 			       (meep-fields-dt-get fields)
 			       (car dt))
-			   (- fcen (/ df 2)) (+ fcen (/ df 2)) maxbands
-                           harminv-spectral-density harminv-Q-thresh
-                           harminv-rel-err-thresh harminv-err-thresh
-                           harminv-rel-amp-thresh harminv-amp-thresh)))
+			   (- fcen (/ df 2)) (+ fcen (/ df 2)) maxbands)))
```

Comment:
Whoops; I must not have have been working on the most recent version of this file when I made this pull request.